app/services/bulk_upload/lettings/year2023/row_parser.rb

```diff
@@ -853,11 +855,12 @@ def validate_if_log_already_exists
       errors.add(:field_7, error_message) # startdate
       errors.add(:field_8, error_message) # startdate
       errors.add(:field_9, error_message) # startdate
+      errors.add(:field_13, error_message) # tenancycode
       errors.add(:field_14, error_message) # propcode
-      errors.add(:field_17, error_message) # location
-      errors.add(:field_23, error_message) # postcode_full
-      errors.add(:field_24, error_message) # postcode_full
-      errors.add(:field_25, error_message) # la
+      errors.add(:field_17, error_message) unless field_4 == 1 # location
+      errors.add(:field_23, error_message) unless field_4 == 2 # postcode_full
```

Comment:
Yeah same again, using != and unless when talking about the same thing. 
I don't know, maybe this is done elsewhere and others are ok with it, but for me I might be tempted to double check the logic when working on this in the future as I'm seeing `==`and `!=` very close together.